backend/copr_backend/background_worker_build.py

```diff
@@ -381,7 +382,25 @@ def _proctitle(self, text):
         self.log.debug("setting title: %s", text)
         self.setproctitle(text)
 
+    def _check_build_interrupted(self):
+        """
+        Should we interrupet a running worker?
+        """
+        return (self._cancel_task_check_request()
+                or self._check_failed_resalloc_ticket())
+
+    @ttl_cache(ttl=60)
```

Comment:
Don't we want to use 5 minutes or something like that?  This is catching rather a rare case, and it's acceptable that a few users wait a bit longer for the resubmit...?

Note we run even >= 200 machines in parallel, ttl=60 means >= 3 requests/s to the resalloc server.